bqplot/pyplot.py

```diff
@@ -118,7 +118,7 @@ def hashable(data, v):
     return True
 
 
-def show(key=None, display_toolbar=True):
+def show(key=None, display_toolbar=False):
```

Comment:
Should we really change this? This will break long-time users assumption that it shows a toolbar by default.

Reply:
Actually, maybe `display_toolbar` should be kept `True` by default and used for setting the figure's `show_toolbar` property?

Reply:
Thinking again about this, this sounds like a good idea by @martinRenou.

Reply:
Should we remove the old and ugly toolbar in this PR as well?

Reply:
i think we should since user can still add this ugly toolbar independently.